libraries/core_libs/network/src/tarcap/packets_handlers/dag_sync_packet_handler.cpp

```diff
@@ -58,30 +58,29 @@ void DagSyncPacketHandler::process(const PacketData& packet_data, const std::sha
   }
 
   uint64_t transactions_count = (*it++).toInt<uint64_t>();
-  SharedTransactions new_transactions;
+  std::vector<std::pair<std::shared_ptr<Transaction>, TransactionStatus>> new_transactions;
   std::string transactions_to_log;
   for (uint64_t i = 0; i < transactions_count; i++) {
-    std::shared_ptr<Transaction> trx;
+    std::pair<std::shared_ptr<Transaction>, TransactionStatus> trx;
 
     try {
-      trx = std::make_shared<Transaction>(*it++);
+      trx.first = std::make_shared<Transaction>(*it++);
```

Comment:
I dont see where are you setting trx.second ?

Reply:
It was missing. Fixed